lexicons/com/atproto/repo/applyWrites.json

```diff
@@ -24,16 +24,22 @@
             "writes": {
               "type": "array",
               "items": {"type": "union", "refs": ["#create", "#update", "#delete"], "closed": true}
+            },
+            "swapCommit": {
+              "type": "string",
+              "format": "cid"
             }
           }
         }
-      }
+      },
+      "errors": [
+        {"name": "InvalidSwap"}
+      ]
     },
     "create": {
       "type": "object",
       "required": ["action", "collection", "value"],
       "properties": {
-        "action": {"type": "string", "const": "create"},
```

Comment:
I found these were not needed since you already have to pass a `$type` on each action due to the union.

Reply:
nice 👍